src/components/cart/price-with-promocode.ts

```diff
@@ -0,0 +1,25 @@
+import { createSpan } from '../../utils/create-elements/create-tags';
+import { formatPrice } from '../../helpers/format-price';
+import { TOTAL_PRICE } from '../../pages/cart/constants';
+
+export function createPriceWithPromocode(span: HTMLSpanElement, originalPrice: number, discountedPrice?: number): void {
+  span.replaceChildren();
+  if (discountedPrice !== undefined && discountedPrice < originalPrice) {
```

Comment:
Can we write like that?
```suggestion
  if (discountedPrice && discountedPrice < originalPrice) {
```

Reply:
Done